.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full-vnc
+
+# Install custom tools, runtimes, etc.
+# For example "bastet", a command-line tetris clone:
+# RUN brew install bastet
+#
+# More information: https://www.gitpod.io/docs/config-docker/
```

Comment:
This Dockerfile does nothing, except changing this repository's default image from `gitpod/workspace-full` to `gitpod/workspace-full-vnc`.

While this could have been achieved by simply adding the new image name in the `.gitpod.yml` file (instead of writing a custom Dockerfile), I don't think it's useful to have a VNC remote desktop for this repository (no graphical UIs or apps to run).